Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes.yml

```diff
@@ -0,0 +1,454 @@
+category: Endpoint
+commonfields:
+  id: Malwarebytes
+  version: -1
+configuration:
```

Comment:
please add parameter to set the first fetch timestamp, that is the time range to fetch threats from on the first fetch

Reply:
This is not needed for our use case as the events coming in are new real time events. we only get events which are happened after the integration is configured. The reason being, if we were to get old rtp or suspicious activity events into demisto, first it will overload demisto with lot of events, next the old events are usually subject to change as scans remove threats and they won't be rtp/ suspicious activity events anymore as they become scan detections. So we dont want to give customers a different source of truth both on demisto and mwb ends.

Reply:
but I'll look into it if this can be better handled and let you know

Reply:
the timestamp is not needed because of the above reason.